### PR TITLE
[PL-47074] Patch for PL 1.24.7 Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-47074](https://harness.atlassian.net/browse/PL-47074)

Adding two RN Summary as requested by engineers:
adding PL-46486 as requested by Sahiba
adding PL-46492 as requested by Nataraja

https://harness.atlassian.net/browse/PL-46485
https://harness.atlassian.net/browse/PL-46492

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot. 

[PL-47074]: https://harness.atlassian.net/browse/PL-47074?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by brian-f-harness** [comment]
@saifeemustafaqharness Please add a RN for https://harness.atlassian.net/browse/PL-43686

File: release-notes/platform.md [hunk 1]
```diff
@@ -103,6 +103,8 @@ The following deprecated API endpoints are longer supported:
 
 - Added support for encrypted assertions in the SAML response. (PL-43353)
 
+- Added support for selecting environment in manual credentials option while creating an Azure key vault connector (PL-46485)
```
Comment (brian-f-harness) [inline release-notes/platform.md:106]:
```suggestion
- Added support for selecting the environment in the manual credentials option while creating an Azure key vault connector. (PL-46485)
```

File: release-notes/platform.md [hunk 2]
```diff
@@ -115,6 +117,10 @@ The following deprecated API endpoints are longer supported:
 
    This item is available with Harness Platform version 1.24.7 and does not require a new delegate version. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
 
+- When linking an SSO group with over 1,000 users, only 1,000 users were syncing in Harness due to a limitation with LDAP groups syncing. (PL-46492) (ZD-56741)
```
Comment (brian-f-harness) [inline release-notes/platform.md:120]:
```suggestion
- When linking an SSO group with over 1,000 users, only 1,000 users were syncing in Harness due to a limitation with LDAP groups syncing. (PL-46492, ZD-56741)
```

File: release-notes/platform.md [hunk 3]
```diff
@@ -115,6 +117,10 @@ The following deprecated API endpoints are longer supported:
...
+
+   Implemented LDAP to perform paginated queries by default for large groups, with a fallback to non-paginated calls, ensuring complete user synchronisation.
```
Comment (brian-f-harness) [inline release-notes/platform.md:122]:
```suggestion
   Implemented LDAP to perform paginated queries by default for large groups, with a fallback to non-paginated calls, ensuring complete user synchronization.
```

**Review by brian-f-harness — DISMISSED** [review]
A couple of minor suggestions, otherwise LGTM

**Review by brian-f-harness — APPROVED** [review]

**Review by pratmit — APPROVED** [review]
LGTM

**Pull request merged**